rubix-common/src/main/java/com/qubole/rubix/common/MetricsConstant.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2016. Qubole Inc
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License. See accompanying LICENSE file.
+ */
+
+package com.qubole.rubix.common;
+
+/**
+ * Created by Abhishek on 2/15/18.
+ */
+public final class MetricsConstant
+{
+  private MetricsConstant()
+  {
+  }
+
+  public static final String CACHED_READ_BYTES = "cached_read_bytes";
```

Comment:
dotted convention is the standard. So this one should be something like `bookkeeper.cachedReadBytes`

Also there is a helper function - `MetricRegistry.name(QueueManager.class, "jobs", "size")` I am not a fan of metrics with fully qualified class name in them. Too verbose. 

We can standardize on `daemon.metricUnit`

Reply:
Will make the change. So we are going to rename cached_read_bytes to cachedReadBytes.